lib/puppet/type/acl/ace.rb

```diff
@@ -322,7 +323,7 @@ def inspect
         if key_value.is_a? Array
           "#{key} => #{key_value}"
         else
-          "#{key} => '#{key_value}'"
+          "#{key} => '#{key_value}'" unless (key == 'id' && key_value.nil?)
```

Comment:
Gross... designed to not emit an empty SID (to match prior behavior).

Might have to scrap adding `id` to the returned hash.